ocaml/testsuite/tests/mixed-blocks/structural_constants.ml

```diff
@@ -0,0 +1,86 @@
+(* TEST
+   flambda2;
+   flags="-extension layouts_beta";
```

Comment:
The new check correctly points out that this test doesn't actually run anything.